04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+In more detail, the common pool is represented by a structure with following characteristics:
+
+\begin{itemize}
+    \item Common Pool Threshold.
+    \item Amount of Resources.
+\end{itemize}
+
+According to the specifications of a Linear Public Goods Game, the agents should be able to perform the following actions:  
+
+\begin{itemize}
+    \item Request Resource from the Common Pool.
+    \item Contribute Resource to the Common Pool. 
+\end{itemize}
+
+And the Common Pool is responsible for two more actions: 
+
+\begin{itemize}
+    \item Mitigate disaster.
+    \item Deplete islands after a disaster.
+\end{itemize}
+
+The common pool threshold is a fixed value and known to islands and its resources will be given to each island based on the severity of the damages.
+
+If the amount of resources in the common pool exceeds the total effect of disaster, then the leftover amount of resources in the common pool will be used for the next round.
+
+Since the goal of each island is to maximise its utility, the following dilemma shows up: islands can free ride by not contributing to the common pool and yet receiving the benefits after a disaster or if they are aware that they would not be depleted by the disaster - since they are far away from the eye of the storm - they might also not contribute to the common pool and thus the rest of the islands would be severely destroyed.
+
+The following example can briefly explain this dilemma:
+
+If the epicentre of disaster is known to the islands and happens to coincide with the location of Island 1, then Island 4 might not contribute at all to the Common Pool since it will not be affected by the disaster. Moreover, Islands 3 and 5 that will not be severely hit might contribute a small amount to the Common Pool or even zero amount (free riding case). Since the rest of the islands might contribute to the Common Pool, then Islands 3 and 5 will be profited by the contribution of others.
+
+importance of prevention and incentivise islands to contribute to the Common Pool is highlighted, thus if the amount of resources in the common pool exceeds the threshold when the disaster happens, then the “power” of disaster towards the islands and common pool will be decreased. It makes sense in real life, as more preparation will result in less damage. The actual damage of disaster is decreased by half.
+
+\subsection{Example Distribution}
+This can be described by the examples in Table~\ref{tab:Demonstration of how the common pool work in different scenarios}, which shows 4 possible scenario:
+
+\begin{itemize}
+    \item Contribution to common pool does not meet threshold and common pool cannot fully mitigate disaster’s damage.
+    \item Contribution to common pool does not meet threshold but common pool can fully mitigate disaster’s damage.
```

Comment:
since theses sentences are very similar except for can/cannot it would make it clearer to put that word in italics